client/jetpack-onboarding/steps/summary-todo-links.jsx

```diff
@@ -0,0 +1,58 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { map } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getEditorNewPostPath } from 'state/ui/editor/selectors';
+
+const TodoLinks = ( { steps } ) =>
+	map( steps, ( { label, url }, stepName ) => (
+		<div key={ stepName } className="steps__summary-entry todo">
+			<a href={ url }>{ label }</a>
+		</div>
+	) );
+
+export const TodoLinksConnected = connect( ( state, { siteId, siteSlug, translate } ) => ( {
```

Comment:
To me exporting those two components here feels a little redundant and unnecessary, when we can just export a single one. For better separation of concerns I'd assume that the Summary step wouldn't need to care about the internal logic of the todo links. In addition, I usually am in favor of the rule "one component per file" and that's not the case here.

Reply:
Indeed. I'll move the `isConnected` check inside the component then.

Reply:
(Might make the implementation slightly more ugly, but the interface nicer, so I guess it'll be worth  it.)